packages/components/addon/components/hds/gutters/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div class={{this.classNames}} style={{this.customSpacing}} ...attributes>
```

Comment:
In Ember this (using an inline style attribute) is considered an anti-pattern for security reasons (that's what I've been told).

Here you can see an early implementation where we used it:
https://github.com/hashicorp/design-system/blob/c9b9f12a1071b29c607e76688a6eada0d4ec289f/packages/components/addon/components/hds/breadcrumb/item.js#L7-L38
https://github.com/hashicorp/design-system/blob/c9b9f12a1071b29c607e76688a6eada0d4ec289f/packages/components/addon/components/hds/breadcrumb/item.hbs#L1

and here you can see how we ended up doing it in the components after that feedback, using a `style` ember helper:
https://github.com/hashicorp/design-system/blob/c9b9f12a1071b29c607e76688a6eada0d4ec289f/packages/components/addon/components/hds/dropdown/index.hbs#L11

Another alternative approach in this case, since you're setting a CSS custom prop, could be to using the `setProperty ` method in JS to sets the CSS prop value directly on the element (under the hood, it creates an inline style attribute via JS).

Reply:
Ok thanks.

I used this: `htmlSafe(--hds-spacing: ${spacing})` which I thought would handle security issues but I'll investigate the items mentioned in your feedback.

Reply:
@didoo I'm trying to use setProperty() but I can't figure out how to access the specific element (div or span parent element) to set the style property. Is there a way to do that?

Reply:
> I used this: htmlSafe(--hds-spacing: ${spacing}) which I thought would handle security issues

I did the same (time ago) but then I was told that it doesn't do what one would expect, so the security issue remains using this approach (that's what I've been told).

> I'm trying to use setProperty() but I can't figure out how to access the specific element 
As far as I remember you have to assign an ID to the element, and then use the ID to get the DOM node, and then use target that DOM node. It may be that there is a more straightforward way (I tried long time ago) but I am not sure.

But why not use the `style` modifier as suggested in this comment?
https://github.com/hashicorp/design-system/pull/538#discussion_r958296215
It does all the heavy lifting for you (it's built on top of `setProperty()`).

Reply:
Thanks. I tried the style modifier first but I couldn't get it to work how I needed. It doesn't validate the value since I'm trying to set a css custom property value.